Packs/CarbonBlackProtect/Integrations/CarbonBlackProtect/CarbonBlackProtect.yml

```diff
@@ -1,7 +1,4 @@
 category: Endpoint
-sectionOrder:
-- Connect
-- Collect
```

Comment:
I will check why this was removed during the contribution process but it shouldn't. Let me know if you need me to revert this.

Reply:
Unsure why those got dropped either.  I've added a commit adding those back.